docs/installation.md

```diff
@@ -105,6 +105,14 @@ configuration YAML file.
 
 Find more information in the [routing rules documentation](routing-rules.md).
 
+### Configure logging
+
+Path to `log.properties` needs to be set via `log.levels-file` JVM options
+like `-Dlog.levels-file=etc/log.properties`.
+
+Find more information in the [logging properties](https://trino.io/docs/current/admin/properties-logging.html)
```

Comment:
```suggestion
Use the `log.*` properties from the [Trino logging properties documentation](https://trino.io/docs/current/admin/properties-logging.html) for further configuration.
```

probably .. but .. those have to also be set as -D parameters... need to explain that.